tests/unit/test_etchosts.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2020 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+
+from mrack.actions.etchosts import EtcHostsUpdater
+from mrack.errors import ConfigError
+from mrack.host import Host
+
+from .utils import copy_as_temp
```

Comment:
Do you think it's okay to use `from unit.utils import copy_as_temp`?

Reply:
Interesting, it actually works. Changed. We might want to fix (in a different) patch a similar thing in the other test.